app/build.gradle.kts

```diff
@@ -148,6 +148,10 @@ dependencies {
     implementation(composeBom)
     globalTestImplementation(composeBom)
 
+    // ecrire aussi sur lib version pour meileur format
+    implementation("io.coil-kt:coil-compose:2.0.0")
+    implementation("androidx.compose.material:material-icons-extended:1.5.0")
```

Comment:
Even though these dependencies are only for icons and minor compose additions, next time do a separate PR for the dependencies.

Reply:
I thought that making a PR for just these two implementations wouldn't have been very useful, but it's true that it's important to do so to properly track the project's progress and get approval from other team members for any additions. I'll make sure to create separate PRs for each change next time!